### DISCUSSION TITLE
I guess CSS is my friend 😬 [Jun 11]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5923] [Discord Threads]
> 🙋‍♂️ **Mike_P**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Océane PUECH**  
#### If you found out the answer, I'm interested !  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1105469550918643724)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
